templates/taxi/manufacturer_list.html

```diff
@@ -1,4 +1,5 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
```

Comment:
The 'crispy_forms_filters' template tag library is loaded here, which is necessary for using the 'crispy' filter on the search form.